### ISSUE TITLE
Tooltip implementation

### ISSUE THREAD

**Issue by juliemturner** [n8design/htwoo#58]
Complete ToolTip implementation

**Comment by gabbsmo** [comment]
Got some requests for this internally. Is there remaining work for tooltips in htwoo-core?

**Comment by StfBauer** [comment]
Yeah that#s not what it is meant hear - title do only work also on certain HTML Elments. The piece missing is the placement of the buttons so that they do not fall offscreen.

So far haven't found any good way to do this.
